pyanaconda/network.py

```diff
@@ -296,6 +298,33 @@ def initialize_network():
     log.debug("Initialization finished.")
 
 
+def write_configuration(payload):
+    """Install network configuration to target system."""
+    fcoe_proxy = STORAGE.get_proxy(FCOE)
+    fcoe_nics = fcoe_proxy.GetNics()
+    fcoe_ifaces = [dev.device_name for dev in get_supported_devices()
```

Comment:
Just want to be sure. Is FCoE from storage module completely moved to the module from UI?

Reply:
Yes, it is.

Reply:
Thanks for the confirmation.